src/main/java/com/emc/mongoose/storage/driver/pravega/kvs/PravegaKVSDriver.java

```diff
@@ -230,21 +240,33 @@ Controller createController(final ClientConfig clientConfig) {
         val nodeConfig = netConfig.configVal("node");
         this.nodePort = nodeConfig.intVal("port");
         val endpointAddrList = nodeConfig.listVal("addrs");
-        val hashingConfig = driverConfig.configVal("hashing");
-        val createRoutingKeysConfig = hashingConfig.configVal("key");
-        val createRoutingKeys = createRoutingKeysConfig.boolVal("enabled");
-        val createRoutingKeysPeriod = createRoutingKeysConfig.longVal("count");
-        this.routingKeyFunc = createRoutingKeys ? new RoutingKeyFunctionImpl<>(createRoutingKeysPeriod) : null;
+        val familyConfig = driverConfig.configVal("family");
+        val createFamilyKeysConfig = familyConfig.configVal("key");
+        val createFamilyKeys = createFamilyKeysConfig.boolVal("enabled");
+        this.allowEmptyFamily = createFamilyKeysConfig.boolVal("allow-empty");
+        val familyKeysAmount = allowEmptyFamily ? createFamilyKeysConfig.longVal("count") + 1 :
```

Comment:
Why should we increment it by 1 in case of empty key family allowance?

Reply:
if empty family is allowed we add one more family key, so that we could treat one value of family key as an empty key. E.g., key-family=0 is basically null key family in that case

Reply:
Seems like an easy way to handle null families without a mess